packages/api-sdk/.speakeasy/gen.yaml

```diff
@@ -19,13 +19,14 @@ generation:
     oAuth2ClientCredentialsEnabled: true
     oAuth2PasswordEnabled: true
 typescript:
-  version: 0.0.6
+  version: 0.0.0
   additionalDependencies:
     dependencies: {}
     devDependencies: {}
     peerDependencies: {}
-  additionalPackageJSON: {}
-  author: Speakeasy
+  additionalPackageJSON:
+    private: true
```

Comment:
note: i had to add `private: true` because the package got published to npm due to the (odd) changesets behavior. (i've deleted it from the registry.)